### PR TITLE
[controller] Static linked nsenter, lsblk and lvm utils

### PR DESCRIPTION
**Pull request by AleksZimin**
## Description

This PR makes several key improvements to our sds-node-configurator:
- We've switched to using statically linked `nsenter`, `lsblk`, and `lvm` utilities, ensuring they work reliably on any host without dependency issues.
- The sds-node-configurator container is now distroless, enhancing security and reducing its footprint.
- Error handling and logging within the sds-node-configurator have been improved, making it more robust and easier to troubleshoot.

## Why do we need it, and what problem does it solve?

These changes address the need for more reliable storage utility execution across diverse environments, minimize potential compatibility issues, and improve the overall security and efficiency of our storage management operations.

## What is the expected result?

- Enhanced reliability and compatibility of storage utility execution.
- A more secure and lightweight sds-node-configurator.
- Improved error handling and logging for better operational insight and troubleshooting.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

### CODE REVIEW

File: templates/agent/daemonset.yaml [hunk 1]
```diff
@@ -67,4 +90,18 @@ spec:
 {{- else if eq .Values.sdsNodeConfigurator.logLevel "TRACE" }}
             value: "4"
 {{- end }}
+        volumeMounts:
+        - mountPath: /opt/deckhouse/bin/sds
```
Comment (apolovov) [inline templates/agent/daemonset.yaml:94]:
```suggestion
        - mountPath: /opt/deckhouse/sds
```

File: images/agent/config/config.go [hunk 2]
```diff
@@ -80,7 +80,7 @@ func getMachineId() (string, error) {
 		args := []string{"-m", "-u", "-i", "-n", "-p", "-t", "1", "cat", "/etc/machine-id"}
 
 		var stdout bytes.Buffer
-		cmd := exec.Command("/usr/bin/nsenter", args...)
+		cmd := exec.Command("/usr/local/bin/flant/nsenter.static", args...)
```
Comment (apolovov) [inline images/agent/config/config.go:83]:
```suggestion
		cmd := exec.Command("/opt/deckhouse/sds/nsenter.static", args...)
```

File: images/agent/pkg/utils/commands.go [hunk 3]
```diff
@@ -428,3 +428,10 @@ func extendArgs(args []string) []string {
 	nsenterArgs := []string{"-t", "1", "-m", "-u", "-i", "-n", "-p"}
 	return append(nsenterArgs, args...)
 }
+
+func lvmStaticExtendedArgs(args []string) []string {
+	nsenterArgs := []string{"-t", "1", "-m", "-u", "-i", "-n", "-p"}
+	lvmStaticBin := []string{"--", "/opt/deckhouse/bin/sds/lvm.static"}
```
Comment (apolovov) [inline images/agent/pkg/utils/commands.go:434]: /opt/deckhouse/sds/ to const.

File: images/static-utils-copier/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    automake \
+    libtool \
+    pkg-config \
+    libdevmapper-dev \
+    libaio-dev \
+    thin-provisioning-tools \
+    git \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN git clone https://sourceware.org/git/lvm2.git /lvm2
```
Comment (apolovov) [inline images/static-utils-copier/Dockerfile:18]: To fox?

**Review by dxavx — APPROVED** [review]

**Review by apolovov — APPROVED** [review]

**Pull request merged**